gen/go/v1/config.pbloader.go

```diff
@@ -229,4 +228,12 @@ func (x *DataCapture) loadFromEnv(prefix string, defaultValues *DataCapture) {
 			x.BodyMaxProcessingSizeBytes = &wrappers.Int32Value{Value: defaultValues.BodyMaxProcessingSizeBytes.Value}
 		}
 	}
+	if rawVals, ok := getArrayStringEnv(prefix + "ALLOWED_CONTENT_TYPES"); ok {
+		for _, val := range rawVals {
+			x.AllowedContentTypes = append(x.AllowedContentTypes, wrappers.String(val))
+		}
+	} else if len(x.AllowedContentTypes) == 0 && len(defaultValues.AllowedContentTypes) > 0 {
```

Comment:
did we initialize `defaultValues.AllowedContentTypes`?

Reply:
From what I can tell that is defined in the go agent repo? 

https://github.com/hypertrace/goagent/blob/main/config/defaults.go

Reply:
ah i see. I see they come in as a parameter into this function.